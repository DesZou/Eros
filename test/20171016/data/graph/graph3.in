5 8 6
4 2 825
3 1 23
4 5 570
3 2 279
3 4 502
5 1 216
3 5 285
2 1 719
10
3 4
5 4
4 2
1 5
5 1
3 5
1 5
5 2
1 4
3 5
