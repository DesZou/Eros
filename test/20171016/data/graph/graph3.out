947
879
873
400
400
331
400
656
617
331
